ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetObject.java

```diff
@@ -88,34 +96,31 @@ public GetObjectResult call() throws Exception {
                 if (SyncUtils.needToSync(helpers, bucketName, filePath, ds3Obj.getName(), false)) {
                     Transfer(helpers, ds3Obj);
                     return new GetObjectResult("SUCCESS: Finished syncing object.");
-                }
-                else {
+                } else {
                     return new GetObjectResult("SUCCESS: No need to sync " + objectName);
                 }
             }
 
             Transfer(helpers, ds3Obj);
             return new GetObjectResult("SUCCESS: Finished downloading object.  The object was written to: " + filePath);
-        }
-        catch(final FailedRequestException e) {
-            if(e.getStatusCode() == 500) {
-                LOG.info(e.getMessage());
-                if (LOG.isInfoEnabled()) {
-                    e.printStackTrace();
-                }
-                throw new CommandException( "Error: Cannot communicate with the remote DS3 appliance.", e);
-            }
-            else if(e.getStatusCode() == 404) {
-                if (LOG.isInfoEnabled()) {
-                    e.printStackTrace();
-                }
-                throw new CommandException( "Error: " + e.getMessage(), e);
-            }
-            else {
-                if (LOG.isInfoEnabled()) {
-                    e.printStackTrace();
-                }
-                throw new CommandException( "Error: Encountered an unknown error of ("+ e.getStatusCode() +") while accessing the remote DS3 appliance.", e);
+        } catch (final FailedRequestException e) {
+            switch (e.getStatusCode()) {
+                case 500:
+                    LOG.info(e.getMessage());
+                    if (LOG.isInfoEnabled()) {
+                        e.printStackTrace();
```

Comment:
Don't print the stack trace.  The logging code will print it once it gets caught later.